Makefile

```diff
@@ -18,7 +17,7 @@ MAKE_ZPKG = $(PANDA_ROOTFS)/make-zpkg
 MAKE_GITHUB_RELEASE = $(PANDA_ROOTFS)/make-github-release.py
 
 BUILD_DIR = $(TOP)/build
-VIVADO_VER = 2022.2
+VIVADO_VER = 2023.2
```

Comment:
I see the example CONFIG suggests to set this variable, if this is a default one, shouldn't it be set with `?=` after including CONFIG the same way we do with `VIVADO_MODE`?

Reply:
Not sure it matters for these, as CONFIG is read after and overwrites them. Probably either way is the same.